libvmi/accessors.c

```diff
@@ -95,6 +95,45 @@ vmi_get_winver(
 #endif
 }
 
+uint16_t
+vmi_get_win_buildnumber(
+    vmi_instance_t vmi)
+{
+#ifndef ENABLE_WINDOWS
+    errprint("**LibVMI wasn't compiled with Windows support!\n");
+    return 0;
+#else
+
+#ifndef ENABLE_JSON_PROFILES
+    errprint("**LibVMI wasn't compiled with JSON profiles support!\n");
+    return 0;
+#endif
+
+    if (vmi->json.handler == NULL) {
```

Comment:
This should be moved to be under ` #ifdef ENABLE_SAFETY_CHECKS `

Reply:
@tklengyel, done!